src/components/Dropdown/ButtonDropdown/index.tsx

```diff
@@ -33,6 +33,7 @@ type Props<T> = {
   messageValidateClass: string;
   validationMessage?: string;
   listPosition: ListPosition;
+  withAppendIcon?: boolean;
```

Comment:
please add this to the doc

Reply:
Hi @kaiomagalhaes . I added it to the Dropdown.stories.mdx, as you can see in the image on the PR description. Is that enough, or I'm missing anything else?